Plugin Cubot/src/main/java/net/simon987/cubotplugin/event/WalkDistanceListener.java

```diff
@@ -0,0 +1,39 @@
+package net.simon987.cubotplugin.event;
+
+import net.simon987.server.event.GameEvent;
+import net.simon987.server.event.GameEventListener;
+import net.simon987.server.game.objects.ControllableUnit;
+import net.simon987.server.game.objects.GameObject;
+import net.simon987.server.logging.LogManager;
+import net.simon987.server.user.UserStats;
+
+public class WalkDistanceListener implements GameEventListener {
+
+    private int count = 0;
+    private int deaths = 0;
+
+    @Override
+    public Class getListenedEventType() {
+        return WalkDistanceEvent.class;
+    }
+
+    @Override
+    public void handle(GameEvent event) {
+        WalkDistanceEvent WalkDistanceEvent = (WalkDistanceEvent) event;
+        GameObject object = WalkDistanceEvent.getSource();
+        if (object instanceof ControllableUnit) {
+            //When cubot dies walk counter resets

```

Comment:
I'd prefer if the walking distance would not reset on death

Reply:
changed